internal/job/restore/restore_test.go

```diff
@@ -171,92 +187,61 @@ func TestRestore_IncrementalBackup_Success(t *testing.T) {
-	//     It's size is the same as the full backup.
-	//   - The restore file. It's size is the same
-	//     as the incremental backup's one.
+	//   - A full backup consists of 2 chunks.
+	//   - An incremental backup, consists of 3 chunks.
+	//   - Restore destination volume filled with random data.
+	//     It's size is the same as the 4 chunks.
 	//
 	// Act:
 	//   Run the restore process about the incremental backup.
 	//
 	// Assert:
 	//   Check if the contents of the restore target file is as follows.
-	//     chunk0, 1: the same as the fake raw.img.
-	//     chunk2: zero filled.
+	//     chunk 0, 1, 2: the same as the incremental backup.
+	//     chunk 3: zero filled.
 
 	// Arrange
-	k8sRepo, rbdRepo, volumeInfo := fake.NewStorage()
-	nlvRepo, finRepo, _ := testutil.CreateNLVAndFinRepoForTest(t)
+	fullBackupVolumeSize := defaultChunkSize * 2
+	incrementalVolumeSize := defaultChunkSize * 3
+	blockDeviceSize := defaultChunkSize * 4
 
-	rawImageChunkSize := uint64(4096)
-
-	fullSnapshotSize := rawImageChunkSize * 2
-	fullSnapshot := rbdRepo.CreateFakeSnapshot(utils.GetUniqueName("snap-"),
-		fullSnapshotSize, time.Now())
-	fullBackupInput := testutil.NewBackupInput(k8sRepo, volumeInfo,
-		fullSnapshot.ID, nil, rawImageChunkSize)
-	fullBackupInput.Repo = finRepo
-	fullBackupInput.KubernetesRepo = k8sRepo
-	fullBackupInput.RBDRepo = rbdRepo
-	fullBackupInput.NodeLocalVolumeRepo = nlvRepo
-
-	incrementalSnapshotSize := rawImageChunkSize * 3
-	incrementalSnapshot := rbdRepo.CreateFakeSnapshot(utils.GetUniqueName("snap-"),
-		incrementalSnapshotSize, time.Now())
-	incrementalBackupInput := testutil.NewBackupInput(k8sRepo, volumeInfo,
-		incrementalSnapshot.ID, &fullSnapshot.ID, rawImageChunkSize)
-	incrementalBackupInput.Repo = finRepo
-	incrementalBackupInput.KubernetesRepo = k8sRepo
-	incrementalBackupInput.RBDRepo = rbdRepo
-	incrementalBackupInput.NodeLocalVolumeRepo = nlvRepo
-
-	// Create a full backup
-	fullBackup := backup.NewBackup(fullBackupInput)
-	err := fullBackup.Perform()
-	require.NoError(t, err)
-
-	// Create an incremental backup
-	incrementalBackup := backup.NewBackup(incrementalBackupInput)
-	err = incrementalBackup.Perform()
-	require.NoError(t, err)
+	cfg := setup(t, &setupInput{
+		// Create an incremental backup
+		enableIncrementalBackup: true,
+		volumeSize: []uint64{
+			fullBackupVolumeSize,
+			incrementalVolumeSize,
+		},
+		blockDeviceSize: []uint64{
+			blockDeviceSize,
+			blockDeviceSize,
+		},
+	})
 
 	// Fill raw.img with random data. Although this file has some data
 	// stored by fake backup process, we can replace them here because we won't
 	// use the original data in the restore process.
-	buf := testutil.FillRawImageWithRandomData(t, nlvRepo.GetRawImagePath(), fullSnapshotSize)
-
-	// Create the restore file
-	restorePath := testutil.CreateRestoreFileForTest(t, incrementalSnapshotSize)
-	rVol := fake.NewRestoreVolume(restorePath)
+	testutil.FillRawImageWithRandomData(
```

Comment:
This function name is weird, because it actually writes data to a block device. Could you rename it to `testutil.FillRandomData` or something?